deliverables/README.md

```diff
@@ -21,48 +21,118 @@ For projects on Code Academy, or other external exercises, please just paste a l
 
 ### Section 1: The Front End
 #### Required - Answer these questions in the deliverables file, and complete the following. 
-What is the role of HTML in a web page?  
+What is the role of HTML in a web page? 
+The "skeleton" of the page
+
 What is the role of CSS in a web page?  
-What is the role of Javascript in a web page?  
+The "skin" that goes over the HTML skeleton
 
+What is the role of Javascript in a web page?  
+Mainly Client-side behavior that HTML and CSS is incapable of
 ---
 
 ### Section 2: HTML and CSS 101
 
 #### meta
 Why do we separate HTML and CSS?  
+Stylistic reasons mainly; effciency when coding
```

Comment:
Separating HTML and CSS does help with efficiency and clean code, but they also have different roles in the website (structure v style) which is another reason they are separated.